packages/components/src/tip/index.js

```diff
@@ -1,24 +1,40 @@
+/** @jsx jsx */
+/**
+ * External dependencies
+ */
+import { jsx } from '@emotion/core';
```

Comment:
@saramarcondes Haiii!!!

Interesting idea! From my experience (in figuring out things in G2...), I personally would avoid the `jsx pragma` based approach. [Theme UI](https://theme-ui.com/guides/jsx-pragma/) offers a similar API.

In practice, I found it to be pretty clunky (and slightly jarring if you're unfamiliar with the workflow).

I believe I understand why this was attempted ❤️ .
My guess was to avoid using `styled` :) - which has a great dev workflow, with a slight trade-off in performance.

I would recommend continuing to use `styled` as your continuing your CSS-in-JS journey within Gutenberg.
It would be consistent with the current (but limited) implementation.

I think introducing the JSX pragma may cause some fragmentation in tidying and refactoring things.